README.md

```diff
@@ -61,7 +61,8 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
  
 ### Key Features <a name="key-features"></a>
 
-- **[Check for errors with linters]**
+- **[A UI interface for a calculator application]**
+- **[Linters, to eliminate errors and better optimization]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Kindly remove the `public/robots.txt` file as it is a boilerplate file that would not be needed in this project and subsequent ones.

Reply:
You highlighted a commit that does not have any errors. I have removed the file.